Frameworks/include/CGPathInternal.h

```diff
@@ -51,4 +55,6 @@ struct CGPathElementInternal : CGPathElement {
 };
 typedef struct CGPathElementInternal CGPathElementInternal;
 
+ID2D1Geometry* _getAndClosePathGeometry(CGPathRef path);
```

Comment:
please something more like:

```
HRESULT _CGPathGetGeometry(CGPathRef path, ID2D1Geometry** pGeometry);
```

---
In reply to: [86264580](https://github.com/Microsoft/WinObjC/pull/1307#discussion_r86264580) [](ancestors = 86264580)